src/overlays/actors/ovl_En_Vase/z_en_vase.c

```diff
@@ -5,6 +5,7 @@
  */
 
 #include "z_en_vase.h"
+#include <../assets/objects/object_vase/object_vase.h>
```

Comment:
with our somewhat recent libultra cleanup pr we dont use <> for includes anymore, only quotes.
We also try to avoid relative paths when possible